src/CasperClient.cpp

```diff
@@ -108,6 +108,18 @@ GetItemResult Client::GetItem(std::string state_root_hash, std::string key,
                                                    paramsJSON);
 }
 
+/// Returns the item in json at the given address with the given key.
+/// FIXME: It is only a workaround, as GetItem function does not parse
+/// correctly result from json:
+/// https://matterfi.atlassian.net/browse/CD-216
+nlohmann::json Client::GetItem_WA(std::string state_root_hash, std::string key,
```

Comment:
why nlohmann ? its just noob question :)

Reply:
Ask creators of casper-cpp-sdk :)

Reply:
It's nice to use. At least for me.
After a documentation:
```Assume you want to create the JSON object:
{
  "pi": 3.141,
  "happy": true,
  "name": "Niels",
  "nothing": null,
  "answer": {
    "everything": 42
  },
  "list": [1, 0, 2],
  "object": {
    "currency": "USD",
    "value": 42.99
  }
}```
With this library, you could write:
```
// create an empty structure (null)
json j;

// add a number that is stored as double (note the implicit conversion of j to an object)
j["pi"] = 3.141;

// add a Boolean that is stored as bool
j["happy"] = true;

// add a string that is stored as std::string
j["name"] = "Niels";

// add another null object by passing nullptr
j["nothing"] = nullptr;

// add an object inside the object
j["answer"]["everything"] = 42;

// add an array that is stored as std::vector (using an initializer list)
j["list"] = { 1, 0, 2 };

// add another object (using an initializer list of pairs)
j["object"] = { {"currency", "USD"}, {"value", 42.99} };

// instead, you could also write (which looks very similar to the JSON above)
json j2 = {
  {"pi", 3.141},
  {"happy", true},
  {"name", "Niels"},
  {"nothing", nullptr},
  {"answer", {
    {"everything", 42}
  }},
  {"list", {1, 0, 2}},
  {"object", {
    {"currency", "USD"},
    {"value", 42.99}
  }}
};
```